pluralsh/plural-artifacts
# feat(datahub): Bump version to 0.13.2

## Summary
Bumps the version of Datahub to a newer version. Not quite the latest but this version bump requires the least amount of changes in terms of dependencies to `elasticsearch` and `kafka`.

I am unsure if the changes in the `Chart.lock` should be committed, as I'm not sure if the SHA will change once plural's CICD packages the charts and duplicates the repositories. Please advise. I am also not sure when the images are pushed to dkr.plural.sh.

Additional reference here: https://artifacthub.io/packages/helm/datahub/datahub/0.4.19

## Test Plan
I tested this change using the following steps:
1. Added helm repo from datahub: https://helm.datahubproject.io
2. Point image repository to datahub's docker registry: https://hub.docker.com/u/acryldata
3. Ensure `helm dependency upgrade` works
4. Use `plural link ...` to link to my local project
5. Run `plural build --only datahub --force && plural deploy`
6. Ensure containers all run happily


## Checklist
- [x]  No images hosted from dockerhub
- [ ]  Are dashboards present to understand the health of the application.  There **must** be at least 1 of these
    - [ ]  all databases should have dashboards
    - [ ]  ideally also have at least cpu/mem utilization dashboards for webserver tier of the app
    - [ ]  you can use `plural from-grafana` to convert a grafana dashboard found via google to our CRD
- [ ]  Are scaling runbooks present
    - [ ]  all databases **must** have scaling runbooks
    - [ ]  you can use the charts in `pluralsh/module-library` to accelerate this
- [ ]  do you need to add config overlays?
    - [ ]  inputing secrets
    - [ ]  configuring autoscaling
- [ ]  If there’s a web-facing component to the app, we need to support OIDC authentication and setting up private networks if no authentication option is viable
- [ ]  All major clouds must be supported
    - [ ]  Azure
    - [ ]  AWS
    - [ ]  GCP
